src/alto-ui/Form/CheckBox/story.js

```diff
@@ -2,15 +2,16 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import centered from '@storybook/addon-centered';
+import { boolean } from '@storybook/addon-knobs';
 
 import CheckBox from './CheckBox';
 
 storiesOf('Form/CheckBox', module)
   .addDecorator(centered)
   .addWithJSX('overview', () => (
     <div>
-      <CheckBox id="unchecked" label="Unchecked" />
-      <CheckBox id="checked" label="Checked" checked />
+      <CheckBox id="unchecked" label="Unchecked" readOnly={boolean('readOnly', false)} />
+      <CheckBox id="checked" label="Checked" defaultChecked readOnly={boolean('readOnly', false)} />
       <CheckBox id="disabled" label="Disabled" disabled />
       <CheckBox id="disabled-checked" label="Disabled checked" disabled checked />
```

Comment:
You should create two new checkboxes. so on the storybook we can test the ones without readOnly.
on checked and other not

Reply:
Fixed